core/drivers/tpm.c

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, ARM Limited
+ */
+
+#include <kernel/generic_boot.h>
+#include <mm/core_mmu.h>
+#include <mm/core_memprot.h>
+#ifdef CFG_DT
+#include <kernel/dt.h>
+#include <libfdt.h>
+#endif
+#include <drivers/tpm.h>
+#include <string.h>
+
+vaddr_t log_va_addr;
+size_t log_size;
```

Comment:
Global variables need a proper prefix. `tpm_log_va` and `tpm_log_size` for instance (note `va_addr` is redundant, "virtual address address").